test/parallel/test-https-simple.js

```diff
@@ -36,14 +36,8 @@ const options = {
   cert: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem')
 };
 
-const tests = 2;
-let successful = 0;
-
 const testSucceeded = function() {
-  successful = successful + 1;
-  if (successful === tests) {
-    server.close();
-  }
+  server.close();
```

Comment:
``` server.close()``` is within the testSucceeded function? and it is only called after this

```
checkCertReq.on('error', function(e) {
	assert.strictEqual(e.code, 'UNABLE_TO_VERIFY_LEAF_SIGNATURE');
    testSucceeded();
})
```
 @ line 100